darwin/dataset/download_manager.py

```diff
@@ -343,8 +353,12 @@ def _download_single_slot_from_json_annotation(
             image = slot.source_files[0]
             image_url = image["url"]
             image_filename = image["file_name"]
-            suffix = Path(image_filename).suffix
-            stem = Path(annotation.filename).stem
+            if image_filename.endswith(".nii.gz"):
+                suffix = ".nii.gz"
+                stem = annotation.filename[: -len(suffix)]
+            else:
+                suffix = Path(image_filename).suffix
+                stem = Path(annotation.filename).stem
```

Comment:
Small extra bug-fix when downloading single-slotted NifTI files (.nii.gz) I noticed when working on this. Because of the way we handle suffixes when downloading single-slotted items, they are currentyl named .gz. NifTI files are a special case of being the only file type we support with 2 extensions. Therefore, we simply handle this case here

This is not a problem for the multi-slotted case, because there we take the entire source name as the filename (which includes .nii.gz)